eng/upload/scripts/UploadPackages.ps1

```diff
@@ -14,7 +14,8 @@ try {
         "Microsoft.DotNet.Interactive.*.nupkg", # note that this also matches Microsoft.DotNet.Interactive.Rendering
         "Microsoft.DotNet.Try.ProjectTemplate.Tutorial.*.nupkg",
         "MLS.Blazor.*.nupkg",
-        "MLS.WasmCodeRunner.*.nupkg"
+        "MLS.WasmCodeRunner.*.nupkg",
+        "WorkspaceServer.nupkg"
```

Comment:
This should be `"WorkspaceServer.*.nupkg"` because the package version is inserted at build time (e.g., `WorkspaceServer.1.2.3.4.nupkg`.

Reply:
You can tell I'm multi-tasking today.